Update content usage rules in `Tournaments/Official_Support` (#11974)
ppy/osu-wiki: wiki/Tournaments/Official_support/en.md

- The forum threads associated with the tournament — this includes forum threads for any preliminary events — **MUST** contain a clearly visible link in a normal-sized font to the [tournament reports form](https://pif.ephemeral.ink/tournament-reports) as the very last content of the original post.
  - `https://pif.ephemeral.ink/tournament-reports`
  - This report form is overseen by the [Tournament Committee](/wiki/People/Tournament_Committee). We encourage all users — players and staff alike — to make use of this form where necessary. Any breaches of [expectations](#expectations), [other procedures](#other-procedures), or [eligibility](#eligibility) requirements should prompt a report.
- The tournament must abide by the following content usage rules:
  - Usage of any beatmaps which have a DMCA takedown notice in any form is strictly prohibited.

>   - The creation or reuse of any beatmaps that are not **Ranked, Approved, or Loved** and violate the [content usage permissions](/wiki/Rules/Content_usage_permissions#artist-permissions) page is strictly prohibited.

Reviewer: violating a page doesn't sound right
reordering to make it easier to understand

```suggestion
  - The creation or reuse of any beatmaps that violate [content usage permission rules](/wiki/Rules/Content_usage_permissions#artist-permissions) is strictly prohibited, except if they are **Ranked, Approved, or Loved**.
```
